app/models/registration.rb

```diff
@@ -240,12 +240,12 @@ def registration_history
                             end
       end
       {
-        changed_attributes: changed_attributes,
+        changed_attributes: changed_attributes.with_indifferent_access,
```

Comment:
Added to make accessing the attributes in tests more consistent

Reply:
Not a big fan of this. What does `index_by` above give you by default, and can you use something like `symbolize_keys` to make it consistent?